wordle-web/components/LeaderBoard2.vue

```diff
@@ -0,0 +1,65 @@
+<template>
+  <v-container>
+    <v-btn @click="refresh">Refresh</v-btn>
+
+    <v-container v-if="!hasData">
+      There are no leaderboard entries at this time...
+    </v-container>
+
+    <v-container>
+        <v-row>
+        <v-col col = "3" justify="center"> Name </v-col>
+        <v-col col = "3" justify="center"> Game Count </v-col>
+        <v-col col = "3" justify="center"> Average Guesses </v-col>
+        <v-col col = "3" justify="center"> Average Seconds/Game </v-col>
+      </v-row>    
+      
+    </v-container>
+
+    <v-container v-if="hasData">
+    <v-row 
+      v-for="t in arr"
+      :key="t"
+      justify="center"
+      >
+      <v-col col="3"  justify="center">
+        {{t.name}}
+      </v-col>
+      <v-col cols="3">
+        {{t.gameCount}}
+      </v-col>
+      <v-col cols="3">
+        {{t.averageGuesses}}
+      </v-col>
+      <v-col cols="3">
+        {{t.averageSecondsPerGame}}
+      </v-col>
+    </v-row>
+    </v-container>
```

Comment:
an alternative to using rows and columns for a table is the v-simple-table, it might make your life easier working with tables in the future
https://vuetifyjs.com/en/components/simple-tables/